### PR TITLE
Fix nil pointer panic when meta bucket is missing

### PR DESCRIPTION
**Pull request by op**
If `NewQuadStore` was called without first calling `InitQuadStore`, the runtime would panic since the meta bucket never was created.

Feel free to close this PR and do it differently. Ran into this bug when I explored cayley for the first time. Hopefully it saves more gopher puppies from running into a nil pointer panic.

I signed the agreement in case.

### CODE REVIEW

**Comment by kortschak** [comment]
s/errBucketNotExist/errNoBucket/g but it seems to me that if you call NewQuadStore without having done the init, a panic is appropriate, maybe just with a better message and not just a nil panic.

**Comment by op** [comment]
Alright. Fixed!

File: graph/bolt/quadstore.go [hunk 1]
```diff
@@ -97,7 +101,9 @@ func newQuadStore(path string, options graph.Options) (graph.QuadStore, error) {
 		return nil, err
 	}
 	err = qs.getMetadata()
-	if err != nil {
+	if err == errNoBucket {
+		panic("bolt: quad has not been initialised")
```
Comment (kortschak) [inline graph/bolt/quadstore.go]: s/quad/quadstore/

**Comment by barakmich** [comment]
LGTM.

**Pull request merged**